frontend/src/metabase/visualizations/lib/apply_axis.js

```diff
@@ -243,6 +249,9 @@ export function applyChartQuantitativeXAxis(
 
     chart.xAxis().tickFormat(d => {
       // don't show ticks that aren't multiples of xInterval
```

Comment:
This comment should be moved down to the next `if`.

Reply:
Ah, good point. I've moved it down.